elephant/utils.py

```diff
@@ -372,3 +394,54 @@ def get_opencl_capability():
         return True
     except ImportError:
         return False
+
+
+def trials_to_list_of_spiketrainlist(method):
+    """
+    Decorator to convert Trials object to a list of spiketrainlists before
```

Comment:
Just small fixes in the docstrings: backticks in Trials, spiketrainlists is not the actual Python classs

Reply:
Fixed.